gradle/libs.versions.toml

```diff
@@ -79,3 +79,51 @@ resteasy          = "6.2.4.Final"
 # opentelemetry dependencies
 opentelemetry         = "1.41.0"
 opentelemetrysemconv  = "1.27.0-alpha"
+
+[libraries]
+hdrhistogram = { group = "org.hdrhistogram", name = "HdrHistogram", version.ref = "hdrhistogram" }
```

Comment:
Avoiding using hyphens and underscores in library or bundle names that are directly referenced outside this file. When using hyphens and underscores, they need to be changed to . when referenced: https://github.com/gradle/gradle/issues/18050